value.go

```diff
@@ -1023,3 +1024,29 @@ func stringToReflectValue(value string, kind reflect.Kind) (reflect.Value, error
 	// FIXME This should end up as a TypeError?
 	panic(fmt.Errorf("invalid conversion of %q to reflect.Kind: %v", value, kind))
 }
+
+// ToStruct will attempt populate a struct passed in as a parameter.
+//
+// ToStruct returns an error if it runs into a problem.
+//
+// Example:
+// a := struct{One int, Two string}{}
+// val, _ := vm.Run(`(function (){ return {One: 1, Two: "two"}}())`)
+// _ := val.ToSruct(&a)
+// fmt.Printf("One: %d, Two: %s\n", a.One, a.Two)
+//
+func (value Value) ToStruct(aStruct interface{}) error {
+	raw, err := value.Export()
+	if err != nil {
+		return fmt.Errorf("failed to export value, %s", err)
+	}
+	src, err := json.Marshal(raw)
```

Comment:
I dunno, it's seems kind of wrong to marshal into json only to immediate demarshal it - like there really should just be a direct marshalling to structs. However, obviously writing that is a lot more work so I can understand reusing the json package to do this, even if the intermediate step of storage as a byte array is an unnecessary annoyance.